SharkCage/CageManager/CageManager.cpp

```diff
@@ -38,13 +65,15 @@ int main()
 
 	if (parse_result != CageMessage::START_PROCESS)
 	{
+		tokenLib::deleteLocalGroup(static_cast<LPWSTR const>(const_cast<wchar_t*>((group_name.c_str()))));
```

Comment:
I think the `const_cast` alone should be sufficient, see a few lines (l. 76) below: `LWPSTR` and `wchar_t *` should be the same